superduperdb/db/base/db.py

```diff
@@ -169,6 +169,53 @@ def show(
             type_id=type_id, identifier=identifier, version=version
         )
 
+    def _get_context(
+        self, model, context_select: t.Optional[Select], context_key: t.Optional[str]
+    ):
+        assert (
+            model.takes_context  # type: ignore[attr-defined]
+        ), 'model does not take context'
+        context = list(self.execute(context_select))  # type: ignore[arg-type]
+        context = [x.unpack() for x in context]
+        if context_key is not None:
+            context = [MongoStyleDict(x)[context_key] for x in context]
+        return context
+
+    async def apredict(
+        self,
+        model: str,
+        input: t.Union[Document, t.Any],
+        context_select: t.Optional[Select] = None,
+        context_key: t.Optional[str] = None,
+        **kwargs,
+    ):
+        """
+        Apply model to input using asyncio.
+
+        :param model_identifier: model or ``str`` referring to an uploaded model
+        :param input: input to be passed to the model.
+                      Must be possible to encode with registered encoders
+        :param one: if True passed a single document else passed multiple documents
```

Comment:
These parameter names are wrong and should be updated. Same as for `predict` below.